app/domain/authentication/authn_oidc/update_input_with_username_from_id_token.rb

```diff
@@ -12,7 +12,7 @@ module AuthnOidc
     ) do
       extend(Forwardable)
       def_delegators(:@authenticator_input, :service_id, :authenticator_name,
-                     :account, :username, :webservice, :credentials, :client_ip,
+                     :account, :username, :webservice, :credentials, :client_ip, :request,
```

Comment:
Isn't the `request` and `credentials` overlapping? The `credentials` value is supposed to be taken from the body of the request, but then we send the entire request as another parameter, which contains the body as well.

Reply:
done

Reply:
@oburstein-hub, avoiding a cascading change feels like the right tradeoff.  Mind adding a comment as to why we're adding the `request` object (so future engineers understand the decision)?

Reply:
added comment